openapi/src/openapi-src-v2.json

```diff
@@ -18,5 +18,214 @@
         }
     ],
     "tags": [{"name": "Questionnaires"}],
-    "paths": {}
+    "paths": {
+        "/questionnaires": {
+            "post": {
+                "tags": ["Questionnaires"],
+                "summary": "Create a new questionnaire from a specified template",
+                "description": "Create a questionnaire",
+                "operationId": "createQuestionnaires",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/apiVersion"
+                    }
+                ],
+                "x-scopes": ["create:questionnaires"],
+                "x-requests": {
```

Comment:
I'm not sure we need these if you're manually writing the tests now? However we still need equivalent manual tests e.g. testing scopes